src/otx/algorithms/visual_prompting/tasks/openvino.py

```diff
@@ -149,11 +152,19 @@ def __init__(
         self.transform = get_transform()  # TODO (sungchul): insert args
 
     def pre_process(
-        self, dataset_item: DatasetItemEntity, extra_processing: bool = False
+        self,
+        dataset_item: DatasetItemEntity,
+        extra_processing: bool = False,
+        use_bbox: bool = False,
+        use_point: bool = False,
     ) -> Tuple[Dict[str, Any], Dict[str, Any], List[Dict[str, Any]]]:
         """Pre-process function of OpenVINO Visual Prompting Inferencer for image encoder."""
+        if use_bbox and use_point:
+            logger.warning("If both use_bbox and use_point are set, bboxes and points will be generated randomly.")
+
+        prob = 1.0 if not use_point else 0.0 if not use_bbox and use_point else 0.5
         images, meta = self.model["image_encoder"].preprocess(dataset_item.numpy, extra_processing)
-        prompts = OTXVisualPromptingDataset.get_prompts(dataset_item, self.labels)  # to be replaced
+        prompts = OTXVisualPromptingDataset.get_prompts(dataset_item, self.labels, prob=prob)
```

Comment:
I suggest to use other flags than prob in the future.
With a given flag, it looks better to choose the mode by random number generation in get_prompts inside.

Reply:
I'll consider that.